src/test/java/org/jenkins/plugins/statistics/gatherer/listeners/RunStatsListenerTest.java

```diff
@@ -45,11 +46,11 @@
     public void setup() {
         listener = new RunStatsListener();
         mockStatic(PropertyLoader.class);
-        mockStatic(RestClientUtil.class);
     }
 
     @Test
     public void givenRunWithBuildInfoTrue_whenStarted_thenPostTwice() throws Exception {
+        mockStatic(RestClientUtil.class);
```

Comment:
Is there a reason why you don't mock it in the setup?

Reply:
Because not all the tests need to mock it. I typically tend to make the test as readable as possible, avoiding mocking stuff that isn't always needed.